### DISCUSSION TITLE
[windows] Which decimal seperator does chia client accept on transaction?

### DISCUSSION THREAD

**Discussion by Plopsi** [Chia-Network/chia-blockchain#5353] [Support]
A fair question? Do I use , or .? Is it depending on OS or is it hardcoded? Which one? Do I get a warning if there are many 0 in game or does it just kick of? So many questions...

And you know the are meaningful, because english it's a decimal point. Others may be commata...

So is 0.000001 the same as 0,000001 or will it be interpreted different, depending on language and OS?

Edit: Sorry, just sow how exceptional demanding this post is. It's late and I wanted to send some mojos, but was unsure. Please forgive the tone of this post :/.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
